File: C/fibonacci.c
#include<stdio.h>
int fib(int a, int b){
	int c= a+b;
	if(c>100000000){
		return c;
	}else{
		printf("%d\n",c);
		return fib(b,c);
	}

}
int main(){
	printf("%d",fib(0,1));
	return 0;
}

File: Rust/fibonacci.rs
fn fib(a:i32,b:i32)->i32{
    let c=a+b;
    if c>100
    {
        return c;
    }
    else
    {
        println!("{}",c);
        return fib(b,c);


    }



}



fn main(){
    println!("{}",fib(0,1));


}
